runners/katacoda/index.ts

```diff
@@ -425,9 +425,10 @@ export class Katacoda extends Runner {
             "interrupt" : terminal ?  terminal.isRunning : false,
             "args": (runCommand.command.parameters.length > 2 && runCommand.command.parameters[2].args) ? runCommand.command.parameters[2].args.join(" ") : undefined
         }
+        let newTerminal = this.terminals.filter(terminal => terminal.terminalId == bashCommand.terminalId).length == 0;
 
         this.pushStep(runCommand, "Executing the command "+ runCommand.command.parameters[1] , "step"+ runCommand.stepIndex + ".md");
-        this.renderTemplate("executeCommand.md", this.outputPathTutorial + "step" + (runCommand.stepIndex) + ".md", { text: runCommand.text, textAfter: runCommand.textAfter, bashCommand: bashCommand});
```

Comment:
I think there is a second bug, that not the relative path from current path is saved in path, but the path relative to workspace. Could you add it too, please?